### ISSUE TITLE
Feature Request: Analogue Pocket GBA Core Support

### ISSUE THREAD

**Issue by CodebyCR** [GearsProgress/Poke_Transporter_GB#10]
Hey there, I love this kind of old Pokemon Game support.
Like many enthusiasts I got an `Analogue Pocket`, which can play my Cardridge and also `.gba` files if you install an GBA core on this.
Anyways, I my opinion this will really fit in the analogue pocket environment.

The idea is to make the flash card unrequited for an Analogue Pocket environment.
So that you can load the Poke_Transporter.gba with the Analogue Pocket GBA core and read (in the bestway) the original Cardridge or another .gba file of the 3.Gen.

**Comment by GearsProgress** [comment]
Hello!! I've gotten a lot of requests about the Analogue Pocket and I definitely want to look into making this process smooth for those users. Unfortunately, I don't own an Analogue Pocket at the moment, so I can't do testing for it. Someday in the near future I'd like to get one, but they're not currently in stock. In the meantime, I'm going to look into the documentation and see how it works with Multiboot titles- it honestly might just work out of the box like that haha

**Comment by Matthiosy** [comment]
Yes, I am also a owner of an Analogue pocket, but not of a flashcard.
I would also love it, if the transporter would correctly run on an emulator like the analogue pocket too, without having to buy a pricey flashcard.
I will be following this progress closely, and hoping for it to work in the future 🙏
My thanks for being willing to look into it and the fantastic work!
